### PR TITLE
Add OpenHub integration

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

**Comment by espadrine** [comment]
Thanks for contributing!

I'm not completely familiar with how OpenHub works. What is the difference between kudo rank and kudo position?

I'd like to have the badges be more intuitive (if I can't tell what the figures mean, I'd probably not be alone), but I'm not sure how to improve it.

Also, it's preferable to have all the data description on the left, and just a figure on the right. The Twitter button, for instance, doesn't have `[ tweet | 42 already tweeted this page ]`, but simply `[ tweet | 42 ]`, and the meaning of the number is easy to infer.

File: server.js [hunk 1]
```diff
@@ -3082,6 +3082,42 @@ cache(function(data, match, sendBadge, request) {
...
+cache(function(data, match, sendBadge, request) {
+  var info = match[1]; // either `kr`, `kp`
+  var user = match[2]; // eg, raphink
+  var format = match[3];
+  var url = 'https://www.openhub.net/accounts/'+user+'.json';
+  var badgeData = getBadgeData('openhub', data);
+  request(url, function (err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(badgeData, format);
```
Comment (espadrine) [inline server.js]: There should be a return statement.

**Comment by olivierlacan** [comment]
@raphink Any news on this, or should we close it?

**Comment by raphink** [comment]
I don't think I'll be working on it

**Comment by paulmelnikow** [comment]
Going to close this then. If anyone would like to pick it up, feel free.

**Pull request closed without merging**